Finance/yahoo_stock_ticker.10m.py

```diff
@@ -242,32 +247,44 @@ def print_stock(s):
         colored_change = color + change_in_percent + RESET
 
     # Remove appending stock exchange symbol for foreign exchanges, e.g. Apple stock symbol in Frankfurt: APC.F -> APC
-    symbol = s['symbol'].split('.')[0]
+    symbol = s['price']['symbol'].split('.')[0]
     # Convert epoch to human readable time HH:MM:SS
-    time = datetime.fromtimestamp(s['regularMarketTime']).strftime('%X')
+    time = datetime.fromtimestamp(
+        s['price']['regularMarketTime']).strftime('%X')
     # Convert float values to decimals with a precision of two
-    fifty_day = '{:.2f}'.format(s['fiftyDayAverage'])
-    two_hundred_day = '{:.2f}'.format(s['twoHundredDayAverage'])
-    fifty_day_change = '(' + '{:.2f}'.format(s['fiftyDayAverageChangePercent'] * 100) + '%)'
-    two_hundred_day_change = '(' + '{:.2f}'.format(s['twoHundredDayAverageChangePercent'] * 100) + '%)'
+    fifty_day = '{:.2f}'.format(s['summaryDetail']['fiftyDayAverage']['raw'])
+    two_hundred_day = '{:.2f}'.format(
+        s['summaryDetail']['twoHundredDayAverage']['raw'])
+
+    regular_market_day_high = s['summaryDetail']['regularMarketDayHigh']['raw']
+    regular_market_day_low = s['summaryDetail']['regularMarketDayLow']['raw']
+    regular_market_day_range = regular_market_day_high - regular_market_day_low
+
+    fifty_two_week_high = s['summaryDetail']['fiftyTwoWeekHigh']['raw']
+    fifty_two_week_low = s['summaryDetail']['fiftyTwoWeekLow']['raw']
+    fifty_two_week_range = fifty_two_week_high - fifty_two_week_low
```

Comment:
These ranges seem to be removed from the API all together so calculating them manually.